cmd/shortener/main.go

```diff
@@ -1,3 +1,40 @@
 package main
 
-func main() {}
+import (
+	"log"
+	"net/http"
+
+	"github.com/IgorGreusunset/shortener/cmd/config"
+	model "github.com/IgorGreusunset/shortener/internal/app"
+	"github.com/IgorGreusunset/shortener/internal/handlers"
+	"github.com/IgorGreusunset/shortener/internal/storage"
+	"github.com/go-chi/chi/v5"
+)
+
+
+
+
+func main() {
+	router := chi.NewRouter()
+
+	db := storage.NewStorage(map[string]model.URL{})
+
+	//Обертки для handlers, чтобы использовать их в роутере
+	PostHandlerWrapper := func (res http.ResponseWriter, req *http.Request)  {
+		handlers.PostHandler(db, res, req)
+	}
+
+	GetHandlerWrapper := func (res http.ResponseWriter, req *http.Request)  {
+		handlers.GetByIDHandler(db, res, req)
+	}
+
+	router.Post(`/`, PostHandlerWrapper)
+	router.Get(`/{id}`, GetHandlerWrapper)
+
+	config.ParseFlag()
+
+	serverAdd := config.Serv
+
+	log.Fatal(http.ListenAndServe(serverAdd, router))
```

Comment:
Стоит тут все же обрабатывать ошибку, так как `log.Fatal()` завершить процесс с кодом 1, что является неуспехом даже в случае если ошибки не было.
```suggestion
	 if err := http.ListenAndServe(serverAdd, router); err != nil {
	     log.Fatalf("Failed to run server: %v", err)
	 }
```